src/django_idom/websocket_consumer.py

```diff
@@ -1,20 +1,24 @@
 """Anything used to construct a websocket endpoint"""
 import asyncio
+import json
+import logging
 from typing import Any
+from urllib.parse import parse_qsl
 
 from channels.generic.websocket import AsyncJsonWebsocketConsumer
 from idom.core.dispatcher import dispatch_single_view
 from idom.core.layout import Layout, LayoutEvent
-from idom.core.proto import ComponentConstructor
+
+from .app_components import get_component, has_component
+
+
+logger = logging.getLogger(__name__)
```

Comment:
Make logger private (`_logger`). Follow this convention across other files as well.